code/modules/client/preference_setup/loadout/loadout_utility.dm

```diff
@@ -15,14 +15,14 @@
 
 /datum/gear/utility/codex
 	display_name = "the traveler's guide to vir"
-	path = /obj/item/weapon/book/codex/lore/vir
+	path = /obj/item/weapon/book/codex //VOREStation Edit
```

Comment:
um, this doesn't seem right

Reply:
It is. See lore_codex/codex.dm.

/obj/item/weapon/book/codex
	var/root_type = /datum/lore/codex/category/main_vir_lore